attributes/security.rb

```diff
@@ -49,13 +49,7 @@
 default['cdap']['security']['ssl_common_name'] = node['fqdn']
 
 if node['cdap']['cdap_site'].key?('kerberos.auth.enabled') && node['cdap']['cdap_site']['kerberos.auth.enabled'].to_s == 'true'
-  include_attribute 'krb5_utils'
-
-  # For keytab creation
-  default['krb5_utils']['krb5_service_keytabs']['cdap'] = { 'owner' => 'cdap', 'group' => 'cdap', 'mode' => '0640' }
-
   default_realm = node['krb5']['krb5_conf']['realms']['default_realm'].upcase
-
```

Comment:
there is still a reference to ``krb5_utils`` a few lines down from here:  https://github.com/caskdata/cdap_cookbook/blob/489ed9875b4d817517f0542dbee3f2ba559977e3/attributes/security.rb#L57